src/components/Comments/Comment.js

```diff
@@ -367,7 +370,11 @@ const Comment = ({
         </View>
       </View>
 
-      <BlockComponent ref={refBlockComponent} screen={'feed_comment_item'} />
+      <BlockComponent
+        ref={refBlockComponent}
+        refresh={refreshComment}
+        screen={'feed_comment_item'}
+      />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `refresh` prop in `BlockComponent` is a good change as it allows the component to refresh its content when necessary. This can be useful in scenarios where the block data changes and the UI needs to reflect these changes.
```diff
+        refresh={refreshComment}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->